package.json

```diff
@@ -87,5 +85,6 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
-  }
-}
+  },
+  "packageManager": "yarn@1.22.22"
```

Comment:
```suggestion

```

Reply:
I think we should keep this and standardize it. There is no downside to including but omitting creates room for error. Why would you prefer it removed?

Reply:
Then its fine I guess but seems unnecessary because my yarn never did that